# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/Document.cpp
# Commit: 055945e043ea
# Full Hash: 055945e043ea3fc7beb243bf9ce2b59e0a571ed0
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2021-04-05 21:28:12
# Description:
#   Bug 1696352 - Handle SetHadLazyLoadImage failure gracefully r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D110803
# ==============================================================================

diff -r ea1b86fec2cb -r 055945e043ea dom/base/Document.cpp
--- a/dom/base/Document.cpp	Mon Apr 05 15:06:57 2021 +0000
+++ b/dom/base/Document.cpp	Mon Apr 05 15:35:51 2021 +0000
@@ -15851,7 +15851,7 @@
   if (!mLazyLoadImageCount) {
     if (WindowContext* wc = GetTopLevelWindowContext()) {
       if (!wc->HadLazyLoadImage()) {
-        MOZ_ALWAYS_SUCCEEDS(wc->SetHadLazyLoadImage(true));
+        Unused << wc->SetHadLazyLoadImage(true);
       }
     }
   }
